File: Check if the number is balanced/Check if the number is balanced.py
class Solution:
    def balancedNumber(self, N):
        l=0
        r=0
        n=len(str(N))

        for i in range(len(str(N))):
            if i<n//2:
                l+=int(N[i])
                
            elif i>n//2:
                r+=int(N[i])

        if l==r:
            return 1
            
        return 0

File: Check if the number is balanced/Check if the number is balanced.cpp
class Solution{
public:	
    bool balancedNumber(string s)
    {
        int n=s.size();
        int mid=n/2;
        long long int s1=0,s2=0;
        for(int i=0;i<mid;i++)
        {
            s1+=(s[i]-'0');
        }
        for(int i=mid+1;i<n;i++)
        {
            s2+=(s[i]-'0');
        }
        return(s1==s2);
 
    }
};